### DISCUSSION TITLE
[Script request]  Lyrion Music Server

### DISCUSSION THREAD

**Discussion by CustomerJFK** [community-scripts/ProxmoxVE#1022] [Request script] [answered]
### Application Name

 Lyrion Music Server

### Website

https://lyrion.org/   &   https://github.com/lms-community/slimserver

### Description

Music Server 

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Pipitapi** [comment]
I'd love to see this in Proxmox VE Helper-Script.

**Comment by tremor021** [comment]
this is one simple .deb install

download: https://downloads.lms-community.org/LyrionMusicServer_v9.0.2/lyrionmusicserver_9.0.2_amd64.deb
install and acces at http://IP:9000

**Comment by Pipitapi** [comment]
Proxmox VE Helper-Script. It's working! Thank you.

**Comment by tremor021** [comment, marked as answer]
Done with #4992
